docs_user/modules/proc_adopting-telemetry-services.adoc

```diff
@@ -14,6 +14,13 @@ This guide also assumes that:
 * Previous Adoption steps completed. MariaDB, the {identity_service_first_ref} and the data plane should be already adopted.
 //kgilliga:Should this procedure be moved after the "Adopting the data plane" chapter?
 
+* Delete Gnocchi endpoint from previous deployment, if it exists
++
+----
+oc rsh openstackclient openstack endpoint list | grep gnocchi
+oc rsh openstackclient openstack endpoint delete gnocchi
```

Comment:
Can we instead add gnocchi to this list? https://github.com/openstack-k8s-operators/data-plane-adoption/blob/6c16b7f29eafc829b2dcaec4c41dac1925f505ab/docs_user/modules/proc_adopting-the-identity-service.adoc#L66 

That should delete both the service and the endpoint (that's how we delete endpoints for the other services).

Reply:
Thank for, I've made changes according to it.